### PR TITLE
Add the failing test for the stripping BOM bug

### PR DESCRIPTION
**Pull request by vkrol**
> Which issue, if any, is this issue related to?

This is the failing test for #4228. 

> Is there anything in the PR that needs further explanation?

No, it's self explanatory.

I am not sure that it is the right place for this test.

### CODE REVIEW

File: system-tests/fix/fix.test.js [hunk 1]
```diff
@@ -72,6 +72,21 @@ describe("fix", () => {
...
+        configFile: systemTestUtils.caseConfig("fix"),
+        fix: true
+      })
+      .then(() => {
+        return pify(fs.readFile)(stylesheetPath, "utf8").then(fileContent => {
+          expect(fileContent.startsWith("\uFEFF")).toBe(true);
+        });
+      });
+  });
+
```
Comment (evilebottnawi) [inline system-tests/fix/fix.test.js]: Can you add new test case instead modify current

Comment (vkrol) [reply system-tests/fix/fix.test.js]: By test case you mean a new CSS file?

Comment (evilebottnawi) [reply system-tests/fix/fix.test.js]: Yes

Comment (vkrol) [reply system-tests/fix/fix.test.js]: Do I need to create another folder under `system-tests` or the current solution will be appropriate?

File: system-tests/fix/fix.test.js [hunk 2]
```diff
@@ -134,3 +134,34 @@ describe("fix", () => {
       });
   });
 });
+
+describe("fix with bom", () => {
```
Comment (hudochenkov) [inline system-tests/fix/fix.test.js]:
```suggestion
describe("fix with BOM", () => {
```

Comment (vkrol) [reply system-tests/fix/fix.test.js]: OK, thanks!

File: system-tests/fix/fix.test.js [hunk 3]
```diff
@@ -134,3 +134,34 @@ describe("fix", () => {
...
+  // eslint-disable-next-line jest/no-disabled-tests
+  it.skip("doesn't strip BOM", () => {
+    return stylelint
+      .lint({
+        files: [stylesheetPath],
+        configFile: systemTestUtils.caseConfig("fix"),
+        fix: true
+      })
+      .then(() => {
+        return pify(fs.readFile)(stylesheetPath, "utf8").then(fileContent => {
```
Comment (vkrol) [inline system-tests/fix/fix.test.js]: I need to replace `pify` here https://github.com/stylelint/stylelint/commit/14a0da663a52da10ca8df9a0183294e64e4f66fe.

Comment (vkrol) [reply system-tests/fix/fix.test.js]: Fixed.

**Pull request merged**